ui/Screen/Project/Source.svelte

```diff
@@ -71,13 +72,21 @@
     );
   };
 
+  let unsubscribe;
```

Comment:
We should leave a comment and/or a follow-up issue that we want to address this concern generally in the app by ways of encoding it in or navigation/routing infrastructure.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/702/commits/082a9b637f19dbe62018a7e15c873f04c7b3a3aa